pkgs/development/python-modules/mysql-connector/default.nix

```diff
@@ -13,6 +13,17 @@ in buildPythonPackage rec {
     sha256 = "1zb5wf65rnpbk0lw31i4piy0bq09hqa62gx7bh241zc5310zccc7";
   };
 
+  patches = [
+    # mysql-connector overrides MACOSX_DEPLOYMENT_TARGET to 11.
+    # This makes the installation with nixpkgs fail. I suspect, that's
+    # because stdenv.targetPlatform.darwinSdkVersion is (currently) set to
+    # 10.12. The patch reverts
+    # https://github.com/mysql/mysql-connector-python/commit/d1e89fd3d7391084cdf35b0806cb5d2a4b413654
+    # I guess this patch might be removed once stdenv.targetPlatform.darwinSdkVersion
```

Comment:
Just a comment: I'm not so sure about that. With the next version bump after that, it will be out of sync again and fail.